span/daq.py

```diff
@@ -43,11 +47,11 @@ def _addOutputChannels(self, task, channels):
             else:
                 task.ao_channels.add_ao_voltage_chan(self.__name + f"{channel}")
     
-    def _addInputChannels(self, task, channels):
+    def _addInputChannels(self, task: dx.task.Task, channels) -> None:
         """
         Add input channels to the DAQ
         """
-        assert self.name, "Name should be set first!"
+        assert self.name, "Name should be set first."

```

Comment:
For consistency with the `_addOutputChannels` method, this should be changed to `assert not (self.name is None), "Name should be set first."`